modules/tinymce/src/plugins/table/main/ts/selection/SelectionTargets.ts

```diff
@@ -1,22 +1,57 @@
+import { Menu, Toolbar } from '@ephox/bridge';
```

Comment:
We shouldn't be using Bridge here as it's public APIs. Please use the types from `PublicApi` in TinyMCE instead.

- [ ] Don't use bridge types here and use public API types